src/Humanizer/Properties/Resources.af.resx

```diff
@@ -118,39 +118,117 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="DateHumanize_SingleSecondAgo" xml:space="preserve">
-    <value>'n sekonde gelede</value>
```

Comment:
You have changed some of the existing values. I trust these are correct as they might break existing expectations.